storage/storage_transfer/aws_request.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2015, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+import logging
+
+import create_client
+
+
+# Edit this template with desired parameters
+# US Pacific Time Zone
```

Comment:
When looking at the date, I can assume that "You have to specify all the dates below in US Pacific Time Zone." or something like that.